.tugboat/config.yml

```diff
@@ -47,7 +47,12 @@ services:
     commands:
       # Commands that set up the basic preview infrastructure
       init:
-        # Install opcache and imagemagick.
+        - sudo apt update
+        # Install imagick
+        - apt-get install -y libmagickwand-dev
+        - echo '' | pecl install imagick
```

Comment:
What's this echo for? Add a comment explaining why?

Reply:
Added comment.